graylog2-server/src/main/java/org/graylog2/rest/resources/system/GrokResource.java

```diff
@@ -130,6 +141,47 @@ public Response bulkUpdatePatterns(@ApiParam(name = "patterns", required = true)
     }
 
     @PUT
```

Comment:
I think this should be a `@POST` because it depends on the `replace` param if this method it idempotent.